docs/ArbitrumBridgeFacet.md

```
@@ -98,7 +98,7 @@ const quoteResult = {
 }
 ```
 
-A detailed explanation on how to use the /quote endpoint and how to trigger the transaction can be found [here](https://apidocs.li.finance/reference/how-to-transfer-tokens).
+A detailed explanation on how to use the /quote endpoint and how to trigger the transaction can be found [here](https://apidocs.li.fi/reference/how-to-transfer-tokens).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)